backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -213,4 +226,26 @@ public static String parseState(String s) {
     }
     return symptomsMap;
   }
+
+  public static StreetAddress parseAddress(
```

Comment:
I think this should live in the AddressValidationService, rather than turning `Translators` into a Frankensteinian static-but-not-actually method bundle.